src/Cookie.php

```diff
@@ -1,14 +1,45 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Yiisoft\Yii\Web;
 
+use DateInterval;
+use DateTimeImmutable;
+use DateTimeInterface;
+use Exception;
+use InvalidArgumentException;
 use Psr\Http\Message\ResponseInterface;
 
+use function array_filter;
+use function array_map;
+use function array_shift;
+use function explode;
+use function implode;
+use function in_array;
+use function preg_match;
+use function preg_split;
+use function strtolower;
+
 /**
  * Cookie helps adding Set-Cookie header response in order to set cookie
  */
 final class Cookie
 {
+    /**
+     * Regular Expression used to validate cookie name
+     * @see https://tools.ietf.org/html/rfc6265#section-4.1.1
```

Comment:
All `@see` annotations with url to RFC should be replaced to `@link`

Reply:
Yep, thanks, will push those changes soon.